pychromecast/quick_play.py

```diff
@@ -0,0 +1,43 @@
+""" Choose a controller and quick play """
+
+from .controllers.youtube import YouTubeController
+from .controllers.supla import SuplaController
+
+
+def quick_play(cast, app_name, data):
+    """
+    CAST_APP_SCHEMA = {
+        vol.Required('app_name', default=""): cv.string,
+        vol.Required('data'): vol.Schema({
+            vol.Required("media_id"): cv.string,
+            vol.Optional("media_type"): cv.string,
+            vol.Optional("enqueue", default=False): cv.boolean,
+            vol.Optional("index"): cv.string,
+            vol.Optional("extra1"): cv.string,
+            vol.Optional("extra2"): cv.string,
+        }),
+    }
```

Comment:
Do a proper docstring instead.

Reply:
Yup, I'll document it.

I wish I could make the schema even better. The extra1 and extra2 keys are hard to keep track of, but I don't see another way of doing this.